scripts/serie_update.py

```diff
@@ -0,0 +1,590 @@
+#!/usr/bin/python
+
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2019 STMicroelectronics.
+# This script define Stm32SerieUpdate class
+# to be used by update_stm32_package.py
+
+import os
+import sys
+import subprocess
+import re
+from pathlib import Path
+
+
+#################################
+# Execute a command from shell
+#
+def os_cmd(cmd):
+    print(cmd)
+    return subprocess.call(cmd, shell=True)
```

Comment:
better use `check_call` and avoid shell usage.

Reply:
I cannot avoid shell usage otherwise some git commands fails to execute.
I tried check_call(), but it need to reformat each string to split command and arguments. So I tried an automatic split() but some complex command still fails.
So I prefer to avoid to rework every string command in the script.
Is it really an issue to keep call()? I didn't succeed to catch the advantage of using check_call( ) instead of call()

Reply:
`check_call` should work fine with `git`, there's probably a bug in the script. `west` uses `git` and `check_call` as an example: https://github.com/zephyrproject-rtos/west/blob/master/src/west/app/project.py#L334